src/metatrain/experimental/nanopet/model.py

```diff
@@ -413,7 +409,46 @@ def forward(
 
         atomic_features_dict: Dict[str, torch.Tensor] = {}
         for output_name, head in self.heads.items():
-            atomic_features_dict[output_name] = head(node_features)
+            if self.atomic_basis_target_info[output_name][
+                "type"
+            ] == "atomic_basis_spherical" and self.atomic_basis_target_info[
+                output_name
+            ]["sample_kind"].startswith("per_pair"):
+                # a two-center target on an atomic basis must be treated specially
+
+                # pass the node features through its head
+                ll_node_features = head[0](node_features)
+
+                # symmetrize the PET edge features and pass through its head
+                if (
+                    self.atomic_basis_target_info[output_name]["sample_kind"]
+                    == "per_pair_sym"
```

Comment:
why is this a different kind of sample?

Reply:
Because in this case the PET features are still one whole block (i.e. we haven't sliced by s2_pi/first_atom_type/second_atom_type yet), and because it is symmetrized the normal samples aren't complete in info: we have "duplicated" samples that carry the index `s2_pi=+/-1`